test/integration/helpers/Env.cs

```diff
@@ -23,19 +23,28 @@ private static void Init()
                     var path = AppDomain.CurrentDomain.BaseDirectory;
                     var sr = new StreamReader(path + "env.json");
                     var jsonString = sr.ReadToEnd();
-                    _env = JsonConvert.DeserializeObject<Dictionary<string, string>>(jsonString);
+                    _env = JsonSerializer.Deserialize<Dictionary<string, JsonElement>>(jsonString, new JsonSerializerOptions
+                    {
+                        PropertyNameCaseInsensitive = true
+                    });
                 }
             }
-            catch
+            catch (JsonException jsonEx)
             {
-                _env = new Dictionary<string, string>();
+                Console.WriteLine($"Error parsing JSON: {jsonEx.Message}");
+                _env = [];
+            }
+            catch (Exception ex)
+            {
+                Console.WriteLine($"Error initializing environment: {ex.Message}");
+                _env = [];
             }
         }
 
-        private static bool IsTrue(string val)
+        private static bool IsTrue(object val)
         {
-            val = val?.ToLower().Trim();
-            return (val == "true") || (val == "1");
+            val = val?.ToString().ToLower().Trim();
+            return val is (object)"true" or (object)"1";
```

Comment:
I would rather put the `or` part into parentheses to explicitly reflect the priority

Reply:
AFAIK is may be used as a type-checking operator or for pattern matching. Strings comparison could be done using Equals

Reply:
VS thinks other wise in regard to the parentheses
![image](https://github.com/user-attachments/assets/09414067-dbe0-4e17-bf63-0d8e0f75dbf6)

Reply:
Strings comparison is now done using Equals